testdata/goldens/go/index.html

```diff
@@ -783,7 +783,7 @@ <h2 id="cloud_google_com_go_storage_BucketHandle" data-uid="cloud.google.com/go/
 </div>
         <div class="markdown level1 conceptual"></div>
         <strong>Examples</strong>
-        <h6 class="notranslate">exists</h6>
+        <div class="notranslate">exists</div>
```

Comment:
This doesn't look right to me. It seems like for this page, `Examples` should be an `h3` since it's under the `h2` `BucketHandle`. Then `exists` should be an `h4`, right?

If someone is using a screen reader, they should be able to jump to the `Examples` header, and to each example name within `Examples`.

Reply:
Makes sense. Modified Examples and some sub-headers to have `h3` and above headers.